packages/component-library/src/utils/auth/credentials.ts

```diff
@@ -0,0 +1,43 @@
+import type { TokenHandler } from "./tokenhandler.types";
+import { AuthClient } from "@alercebroker/http-client";
+import type { UserSchema } from "@alercebroker/http-client/build/main/lib/clients/authentication/AuthClient.types";
+import { verifySession as genericVerifySession } from "./common";
+
+//To-do implement neverthrow
+export function credentialsAuthFactory(
+  tokenHandler: TokenHandler<{ access: string; refresh: string }>,
+  baseUrl?: string
+) {
+  const config = {
+    baseUrl,
+    withCredentials: true,
+    headers: {
+      "Content-Type": "application/json",
+    },
+  };
+
+  console.log("credentials baseurl", baseUrl);
```

Comment:
keep in mind deleting this at some point

Reply:
yessir